### PR TITLE
Bug/ios fix

### PR DESCRIPTION
**Pull request by SencyNetanel**

### CODE REVIEW

File: ios/Pods/Manifest.lock [hunk 1]
```diff
@@ -1,15 +1,56 @@
 PODS:
   - Flutter (1.0.0)
+  - flutter_smkit_ui (0.1.0):
```
Comment (SencyCaspit) [inline ios/Pods/Manifest.lock:3]: @SencyNetanel Are those items needed to be tracked? or can we add it to .gitignore?

File: ios/Pods/Target Support Files/Pods-RunnerTests/Pods-RunnerTests.debug.xcconfig [hunk 2]
```diff
@@ -1,5 +1,15 @@
 CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = NO
+EXCLUDED_ARCHS[sdk=iphonesimulator*] = arm64
+FRAMEWORK_SEARCH_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/SwiftyJSON" "${PODS_CONFIGURATION_BUILD_DIR}/ZIPFoundation" "${PODS_CONFIGURATION_BUILD_DIR}/flutter_smkit_ui" "${PODS_CONFIGURATION_BUILD_DIR}/integration_test" "${PODS_ROOT}/SMBase" "${PODS_ROOT}/SMBaseUI" "${PODS_ROOT}/SMKit" "${PODS_ROOT}/SMKitUI" "${PODS_XCFRAMEWORKS_BUILD_DIR}/SMBase" "${PODS_XCFRAMEWORKS_BUILD_DIR}/SMBaseUI" "${PODS_XCFRAMEWORKS_BUILD_DIR}/SMKit" "${PODS_XCFRAMEWORKS_BUILD_DIR}/SMKitUI"
```
Comment (SencyCaspit) [inline ios/Pods/Target Support Files/Pods-RunnerTests/Pods-RunnerTests.debug.xcconfig:3]: @SencyNetanel needed to be tracked?

**Review by SencyCaspit — APPROVED** [review]
PR is approved, but left comments about tracked files.

File: ios/Pods/Manifest.lock (hunk 1 above)
Comment (SencyNetanel) [reply ios/Pods/Manifest.lock:3]: those items needed to be tracked

File: ios/Pods/Target Support Files/Pods-RunnerTests/Pods-RunnerTests.debug.xcconfig (hunk 2 above)
Comment (SencyNetanel) [reply ios/Pods/Target Support Files/Pods-RunnerTests/Pods-RunnerTests.debug.xcconfig:3]: those items needed to be tracked

**Pull request merged**